Server/SQL/schema.sql

```diff
@@ -0,0 +1,63 @@
+CREATE TABLE IF NOT EXISTS Card
+(
+    id INTEGER NOT NULL AUTO_INCREMENT PRIMARY KEY,
+    deck_id VARCHAR(255) NOT NULL,
+    needs_review BOOLEAN NOT NULL DEFAULT FALSE,
+    front VARCHAR(255) NOT NULL,
+    back VARCHAR(255) NOT NULL,
+    position INTEGER NOT NULL
+
+    FOREIGN KEY (deck_id) REFERENCES Deck(uuid)
```

Comment:
I don't think we actually will ever be deleting the actual deck ID, will we? But yeah, I can add it just in case.